### ISSUE TITLE
Приостановка админки

### ISSUE THREAD

**Issue by OkyHp** [SB-MaterialAdmin/NewServer#10]
Добавьте возможность приостановить работу админки ( поставить на паузу ).

**Comment by Floower** [comment]
Да, нужная вещь. Иногда админы уезжают, приходится выдавать предупреждения и запоминать срок админки.

**Comment by CrazyHackGUT** [comment]
Данный Issue имело смысл добавить в репозиторий веба, ибо необходимы правки в структуре БД.
Мне лично идея понравилась, реализую. Ждите ответа @Grey-rus.
